ep/build-velox/src/get_velox.sh

```diff
@@ -2,8 +2,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=main
+VELOX_REPO=https://github.com/yma11/velox.git
+VELOX_BRANCH=ut-fix
```

Comment:
@yma11 There's the minor commit in your local branch, can you please PR this to oap-project/velox?